pkg/config/tlscfg/options.go

```diff
@@ -90,6 +90,13 @@ func (p Options) loadCertPool() (*x509.CertPool, error) {
 		if err != nil {
 			return nil, fmt.Errorf("failed to load SystemCertPool: %w", err)
 		}
+		if certPool == nil {
```

Comment:
is this condition specific to Windows? Maybe this whole additional logic should be scoped to windows builds only?